src/components/event/tables/VerificationTable.tsx

```diff
@@ -124,6 +124,7 @@ class VerificationTableBase extends React.Component<Props, State> {
 
 		this.resizeObserver = new ResizeObserver(() => this.getHiddenColumns());
 		this.resizeObserver.observe(this.rootRef.current as HTMLDivElement);
+		this.setExpandStatus(false);
```

Comment:
We have a `getVerificationTablesNodes` function which creates nodes for verification table in helpers/table,
let's change default isExpanded status there. Then we can delete this line